blas/gonum/level1double.go

```diff
@@ -326,122 +326,121 @@ func (Implementation) Drotg(a, b float64) (c, s, r, z float64) {
 // http://www.netlib.org/lapack/explore-html/df/deb/drotmg_8f.html
 // for more details.
 func (Implementation) Drotmg(d1, d2, x1, y1 float64) (p blas.DrotmParams, rd1, rd2, rx1 float64) {
-	var p1, p2, q1, q2, u float64
-
 	const (
 		gam    = 4096.0
-		gamsq  = 16777216.0
-		rgamsq = 5.9604645e-8
+		gamsq  = gam * gam
+		rgamsq = 1.0 / gamsq
```

Comment:
It's not clear this should change. If the goal is to make it more like Fortran, rgamsq is not the same here because the Fortran rounds poorly.

Reply:
The goal is not to be more like the Fortran, since it is demonstrably wrong. I started with that as a way to identify where the problems were.

Reply:
Great.